webapp/src/components/card/subscription/index.tsx

```diff
@@ -61,10 +62,29 @@ const SubscriptionCard = ({handleDeleteSubscrption, subscriptionDetails: {channe
                     labelIconClassName={'icon icon-account-outline icon-label'}
                     value={`Subscription created by ${createdBy}`}
                 />
-                {/* TODO: Below UI will change once Figma is ready*/}
-                <LabelValuePair
-                    value={`Filter(s): Repository - ${repository.slice(0, 3)}, Target Branch - ${targetBranch}`}
-                />
+                {
+                    (repositoryName || targetBranch) && (
+                        <div className='d-flex align-item-center margin-left-5'>
+                            <div className='card-filter'>
+                                <SVGWrapper
+                                    width={14}
+                                    height={14}
+                                    viewBox='0 0 12 12'
+                                >
+                                    {pluginConstants.SVGIcons.cardFilter}
+                                </SVGWrapper>
+                            </div>
+                            <div className='card-chip-wrapper'>
+                                {
+                                    repositoryName && <Chip text={`Repository is ${repositoryName}`}/>
+                                }
+                                {
+                                    targetBranch && <Chip text={`Target Branch is ${targetBranch}`}/>
+                                }
```

Comment:
```suggestion
                                {
                                    <Chip text={ repositoryName  
                                              ? `Repository is ${repositoryName}` 
                                              : `Target Branch is ${targetBranch}`
                                          }
                                    />
                                }
```

Reply:
If both `repositoryName` and `targetBranch` are present at a time then we need to show both of them, so we need two separate `Chip`. If we use ternary then only one of the two will be shown at a time.

Reply:
you can see attached screenshot for this UI case